plugin/adsk/scripts/mayaUsd_createStagesFromExistingLayer.mel

```diff
@@ -71,19 +86,99 @@ proc doCreateStage(string $fileName) {
     else
         $baseName = "UsdStage";
 
-    global string $stageFromExistingLayer_primPath;
-    global string $stageFromExistingLayer_excludePrimPath;
-    global int $stageFromExistingLayer_loadPayloads;
+    string $ppath = `optionVar -q stageFromExistingLayer_primPath`;
+    string $exppath = `optionVar -q stageFromExistingLayer_excludePrimPath`;
+    int $loadp = `optionVar -q stageFromExistingLayer_loadPayloads`;
 
     string $shapeNode = `createNode "mayaUsdProxyShape" -skipSelect -name ($baseName+"Shape")`;
     setAttr -type "string" ($shapeNode+".filePath") $fileName;
-    setAttr -type "string" ($shapeNode+".primPath") $stageFromExistingLayer_primPath;
-    setAttr -type "string" ($shapeNode+".excludePrimPaths") $stageFromExistingLayer_excludePrimPath;
-    setAttr ($shapeNode+".loadPayloads") $stageFromExistingLayer_loadPayloads;
+    setAttr -type "string" ($shapeNode+".primPath") $ppath;
+    setAttr -type "string" ($shapeNode+".excludePrimPaths") $exppath;
+    setAttr ($shapeNode+".loadPayloads") $loadp;
     connectAttr time1.outTime ($shapeNode+".time");
     select -r $shapeNode;
 }
 
+global proc createUSdStagesFromExistingCallback(string $parent, int $doIt)
+{
+    stageFromExistingLayer_UICommit($parent);
+
+	if ($doIt) {
+		mayaUsd_createStagesFromExistingLayer; 
+		addToRecentCommandQueue "mayaUsd_createStagesFromExistingLayer" 
+            "mayaUsdCreateStageFromExistingLayer";
+	}
+}
+
+global proc createUSdStagesFromExistingSetup(string $parent, int $forceFactorySettings)
+{
+    setOptionVars($forceFactorySettings);
+
+    stageFromExistingLayer_UIInit($parent, "");
+}
+
+global proc mayaUsd_createStagesFromExistingLayerOptions()
+{
+    global int $gOptionBoxTemplateFrameSpacing;
+    
+	setOptionVars(false);
+	
+	string $commandName = "createUSdStagesFromExisting";
+	string $callback = ($commandName + "Callback");
```

Comment:
Looks like some inconsistent whitespacing through here.

Reply:
Yeah, looks like I bounced between editors that had different settings for tabs/spaces.  I'll clean it up.